### PR TITLE
Upgrade Rust to 1.59.0 in docker image

### LINKED ISSUE #47: Fails to parse Cargo.toml with `strip` is used
**Describe the bug**

My Cargo.toml contains the following section

```toml
[profile.release]
lto = true
codegen-units = 1
strip = true
```

in GitHub Actions, use of the cargo-deny-action throws the following error:

```
Error: -03 17:04:24 [ERROR] `cargo metadata` exited with an error: error: failed to parse manifest at `/github/workspace/Cargo.toml`

Caused by:
  feature `strip` is required

  The package requires the Cargo feature called `strip`, but that feature is not stabilized in this version of Cargo (1.58.0 (f01b232bc 2022-01-19)).
  Consider trying a newer version of Cargo (this may require the nightly release).
  See https://doc.rust-lang.org/nightly/cargo/reference/unstable.html#profile-strip-option for more information about the status of this feature.
```

**Expected behavior**

I expected the `cargo-deny` action to run without error

**Device:**
 - ubuntu-latest

### PR DESCRIPTION
**Pull request by EricCrosson**
Closes #47

### Checklist

* [X] I have read the [Contributor Guide](../../CONTRIBUTING.md)
* [X] I have read and agree to the [Code of Conduct](../../CODE_OF_CONDUCT.md)
* [X] I have added a description of my changes and why I'd like them included in the section below

### Description of Changes

Upgrade Rust to 1.59.0 in the base image of the Dockerfile to close #47 

### Related Issues

Resolves #47

### CODE REVIEW

**Comment by EricCrosson** [comment]
Howdy -- neat project! Thanks for your work maintaining it

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM rust:1.58.1-alpine3.15
```
Comment (EricCrosson) [inline Dockerfile:1]: https://hub.docker.com/layers/rust/library/rust/1.59.0-alpine3.15/images/sha256-b57e3379714cf4f784ce1de9e6143eda24b385085b94e7f3a4ff115d71ee986c?context=explore

**Comment by Jake-Shadle** [comment]
Closed in https://github.com/EmbarkStudios/cargo-deny-action/releases/tag/v1.2.13, you can now specify which cargo version you want to use. Thanks for the PR though!

**Comment by EricCrosson** [comment]
Oh yep that's a way better approach. Good thinking!

**Pull request closed without merging**